src/main/java/uk/ac/cam/cl/dtg/isaac/dos/IsaacEventPage.java

```diff
@@ -15,212 +15,211 @@
  */
 package uk.ac.cam.cl.dtg.isaac.dos;
 
-import java.util.Date;
-import java.util.List;
-import java.util.Set;
-
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
-
-import uk.ac.cam.cl.dtg.isaac.dto.IsaacEventPageDTO;
 import uk.ac.cam.cl.dtg.isaac.dos.content.Content;
 import uk.ac.cam.cl.dtg.isaac.dos.content.ContentBase;
 import uk.ac.cam.cl.dtg.isaac.dos.content.DTOMapping;
 import uk.ac.cam.cl.dtg.isaac.dos.content.ExternalReference;
 import uk.ac.cam.cl.dtg.isaac.dos.content.Image;
 import uk.ac.cam.cl.dtg.isaac.dos.content.JsonContentType;
+import uk.ac.cam.cl.dtg.isaac.dto.IsaacEventPageDTO;
 import uk.ac.cam.cl.dtg.util.locations.Address;
 import uk.ac.cam.cl.dtg.util.locations.Location;
 
-import static uk.ac.cam.cl.dtg.segue.api.Constants.EVENT_GROUP_RESERVATION_DEFAULT_LIMIT;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+
+import static uk.ac.cam.cl.dtg.segue.api.Constants.*;
```

Comment:
So the Google checkstyle says no star imports ever, but I much prefer
```
import static uk.ac.cam.cl.dtg.segue.api.Constants.*;
import static uk.ac.cam.cl.dtg.isaac.api.Constants.*;
```
over anything else. There can only be collisions if we create them (since only those two classes are allowed to be * imports), so it makes the code much cleaner. I've used this style consistently for years; so chunks of the code will have these two imports like this already, rather than explicit individual imports for these classes.

Do you want to make the case for no starred imports at all? :)

Reply:
It just made me wonder what the new general policy for wildcard imports is, but that makes sense.